pkg/eventing/v1/client.go

```diff
@@ -387,6 +386,80 @@ func (b *BrokerBuilder) Class(class string) *BrokerBuilder {
 	return b
 }
 
+// DlSink for the broker builder
+func (b *BrokerBuilder) DlSink(dlSink *duckv1.Destination) *BrokerBuilder {
+	empty := duckv1.Destination{}
+	if dlSink == nil || *dlSink == empty {
+		return b
```

Comment:
For the sake of managing test coverage, all those corner cases would ideally be covered. IMO `nil` cases should be fine for us.

Reply:
> IMO nil cases should be fine for us.

I used a very confusing construct in my first comment, sorry about that :)

It should really go like this. Let's add test cases with `nil` values to cover return of builder without any changes. It's for the sake of coverage mainly, but it is what it is. :)

Reply:
I added the `nil` test cases.